AUTHORS

```diff
@@ -0,0 +1,4 @@
+Juergen Boehringer (see www.boehri.de)
+Reinhard X. Fuerst (see feinstaub.rexfue.de)
+Thomas Waldmann <twaldmann@thinkmo.de>
```

Comment:
I hope this is complete / correct.

I only used information which is publicly available anyway.